getting-started/6_use_app.md

```diff
@@ -0,0 +1,22 @@
+You should now see the login page for the social network. For simplicity of this app, there is no password or sign-up required. First enter your name and click Log in.
+
+![Login Screen](assets/create-daml-app-login-screen.png)
```

Comment:
![Screenshot from 2020-04-17 13-18-46](https://user-images.githubusercontent.com/1313584/79563942-0254d400-80ae-11ea-8e06-37afb2a6ee56.png)
I broke it (clicking on the login button again doesn’t change anything), no idea what I did wrong but if I manage to break it our users will probably too.

Reply:
Interesting. I ran into this once, but couldn't reproduce. It looked like a bug in `create-daml-app` to me since the network tab looked like it was all working just fine. Maybe @martin-drhu-da or @rohanjr could see whether they can reproduce and debug.

Reply:
not sure this is useful information but I’ve never seen this locally and I’ve gone through the GSG quite a few times recently :)